docs/en/cloud/AWS-CFT.md

```diff
@@ -0,0 +1,261 @@
+  - **WorkerSpotPrice** is an optional field to launch worker instances as spot instances,
+  specifying the maximum hourly price that you are willing to pay for spot instances.
+  Note that the price should be set according to the worker instance type.
+  If the price is too low, Alluxio workers may not be fully provisioned and the stack will show as `CREATE_FAILED`.
+  By default, all instances will be launched [on demand](https://aws.amazon.com/ec2/pricing/on-demand/).
+  EC2 instances can also be launched as [spot instances](https://aws.amazon.com/ec2/spot/).
+  This saves a significant portion of the instance cost,
+  but at the risk of having them terminated and reclaimed by EC2 at any time.
+  Alluxio masters are critical to the cluster and should not be launched as spot instances.
+  In contrast, worker instances fit the use case of spot instances
+  because new workers can register themselves to the cluster and unavailable workers will be marked as lost.
+  Because the addition and loss of workers do not affect basic Alluxio functionality,
+  we can support launching Alluxio clusters with spot instances for workers.
+
+- Alluxio Configuration
+  - **EnableHA** defaults to `No`, which instructs the cluster to launch a single master.
+  If set to `Yes`, three master instances will be provisioned.
+  - **WorkersCount** determines the number of Alluxio workers for the cluster.
+  - **AlluxioRootMountS3BucketName** and **AlluxioRootMountS3BucketPath** are combined form a S3 URI to serve as Alluxio filesystem's root mount.
+  The current user account must have the read/write/list permissions to this S3 URI and the URI will be mounted to the root of the Alluxio file system.
+  - **OpenS3Access** should be set to `Yes` if Alluxio or other services in the cluster will need to access S3 buckets,
+  other than the one defined for the root mount.
+  - **AlluxioProperties** can be set to provide additional Alluxio site properties.
+  Alluxio CFT only provides the necessary parameters to create an Alluxio cluster.
+  If you desire to fine-tune Alluxio behavior, specify the desired properties in the format of `KEY1=VALUE1,KEY2=VALUE2`.
+  The specified key-value pairs will be appended to the `alluxio-site.properties` file in all nodes inside the Alluxio cluster.
+
+- Advanced Alluxio Configuration: These optional parameters are only available in the advanced template.
+  - **AlluxioJournalSize** determines the EBS volume size, in GB, where Alluxio's embedded journal will write to.
+  Each Alluxio master mounts a EBS volume dedicated for logging all metadata changes to achieve fault tolerance.
+  The journal size should be proportional to the estimated magnitude of Alluxio metadata operations.
+  - **AlluxioWorkerMemPercent** is the percentage of the instance's total memory to allocate for the Alluxio worker process.
```

Comment:
This memory is not allocated for the Alluxio worker process, but use as the Alluxio worker storage to store Alluxio cached data. 
So may be remove the process? or change it to storage.